### PR TITLE
[13.0][PORT]stock_picking_show_linked: Backport to 13.0

### PR DESCRIPTION
**Pull request by AnnaPForgeFlow**
@ForgeFlow
Standard backport to 13.0 from 14.0.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@AnnaPForgeFlow Thanks for this.

Could you follow migration guide and preserve commit history ? Thanks

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

**Comment by MiquelRForgeFlow** [comment]
@AnnaPForgeFlow But just have to do this instead:

`git format-patch --keep-subject --stdout origin/13.0..origin/14.0 -- stock_picking_show_linked | git am -3 --keep`

**Comment by rousseldenis** [comment]
> @AnnaPForgeFlow But just have to do this instead:
> 
> `git format-patch --keep-subject --stdout origin/13.0..origin/14.0 -- stock_picking_show_linked | git am -3 --keep`

Yes, and install pre-commit locally before committing

File: stock_picking_show_linked/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "summary": """
+       This addon allows to easily access related pickings
+       (in the case of chained routes) through a button
+       in the parent picking view.
+    """,
+    "version": "13.0.1.0.1",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
+    "depends": ["stock",],
```
Comment (MiquelRForgeFlow) [inline stock_picking_show_linked/__manifest__.py]:
```suggestion
    "depends": ["stock"],
```
flake8

File: stock_picking_show_linked/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        for record in self:
+            origin_pickings = record.mapped("move_lines.move_orig_ids.picking_id")
+            dest_pickings = record.mapped("move_lines.move_dest_ids.picking_id")
+            record.origin_picking_count = len(origin_pickings)
+            record.dest_picking_count = len(dest_pickings)
+
+    def _get_action_link(self, pickings):
+        result = self.env["ir.actions.act_window"].for_xml_id(
+            "stock",
+            "action_picking_tree_all"
```
Comment (MiquelRForgeFlow) [inline stock_picking_show_linked/models/stock_picking.py]:
```suggestion
            "stock", "action_picking_tree_all"
```

**Review by MiquelRForgeFlow — COMMENTED** [review]
whit these, pre-commit errors are solved

**Comment by rousseldenis** [comment]
> whit these, pre-commit errors are solved

@MiquelRForgeFlow @AnnaPForgeFlow As I said, installing pre-commit locally before committing should solve your problems

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by albariera — APPROVED** [review]
LGTM 👍

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**